libzotero/libzotero.py

```diff
@@ -26,6 +26,33 @@
 import time
 from libzotero.zotero_item import zoteroItem as zotero_item
 
+def decode_zotero_str(raw, max_iters=3):
```

Comment:
Sounds very reasonable!

To answer your question: My fix was based on the [observation](https://github.com/kostrykin/qnotero/commit/8e0cec16fe53611bb802d1e8d39788933e4be07d#diff-839e045cb869ebe7428068bf7016edfeR29) that Zotero encoded `'Öf'` (a two-letters string, where the first letter is "Ö", that is a non-ASCII)  as the four-bytes sequence  `\xc3\x83\xc2\x96\x66`. This is strange, because UTF-8 encodes ASCII letters as single bytes, and the letter `f` is clearly ASCII. Empirically, I ruled out the possibility of that being a UTF-16 encoding. So why are two letters encoded as four bytes, when one character is ASCII?

```python
In [14]: '\xc3\x83\xc2\x96\x66'.decode('utf-8')
Out[14]: u'\xc3\x96f'
```

Turns out, that `\xc3\x83\xc2\x96\x66` is the UTF-8 encoding for the three-letters-string `\xC3\x96f` (note the `f` at the end). The first two bytes `\xC3\x96` are *in turn* the UTF-8 representation of `Ö`. So I suspect that this is an odd behaviour from Zotero.